src/software/thunderscope/robot_diagnostics/robot_view.py

```diff
@@ -133,13 +134,10 @@ def refresh(self) -> None:
         Until the buffer is empty
         """
         robot_status = self.robot_status_buffer.get(block=False, return_cached=False)
+        round_trip_time = self.round_trip_time_buffer.get(block=False, return_cached=False)
```

Comment:
this name should also update to reflect that this is the receive time and not RTT